src/Social/Distribution/Distribution.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\Distribution;
+
+use Kanvas\Social\Channels\Models\Channel;
+use Kanvas\Social\Messages\Models\Message;
+
+class Distribution
```

Comment:
@FredPeal chsngr the name to the distribution type

Reply:
Why Distribution type ?

Reply:
cause i dont understand what this is by looking at the name, this is within the domain distribution so, this is referring to feed distribution